tower-http/src/lib.rs

```diff
@@ -1,3 +1,83 @@
+//! `async fn(HttpRequest) -> Result<HttpResponse, Error>`
+//!
+//! # Overview
+//!
+//! tower-http is a library that provides HTTP specific middlewares and utilities built on top of
+//! [tower].
+//!
+//! All middlewares uses the [http] and [http-body] crates as the HTTP abstractions. That means
+//! they're compatible with any library or framework that also uses those crates, such as [hyper].
+//! Some middlewares might bring in other dependencies for doing various things.
```

Comment:
this feels a little bit vague...i'm not sure if it's _really_ necessary to have this line?

Reply:
Yeah I agree. I've removed it.